client/src/components/QuestionsAndAnswers/QAList/index.jsx

```diff
@@ -1,12 +1,17 @@
 import React from 'react';
 import QAListItem from './QAListItem';
 
-const QAList = () => {
+const QAList = ({questionsList, displayedQuestionsList, loadMoreQuestionsClickHandler}) => {
+  const[displayCount, setDisplayCount] = React.useState(displayedQuestionsList?.length); //might not need this state
+  const[isScrollable, setIsScrollable] = React.useState(false); //will beter determine this when i implement the scrollbar
 
   return (
     <div>
       <div>QAList</div>
-      <QAListItem />
+      {questionsList.map((question) => (
+        <QAListItem question={question}/>
```

Comment:
Consider adding a key prop here with the question id to aid React in rendering this.